add-domain

```diff
@@ -747,3 +747,11 @@ soclaiemx.xyz
 wiflix.xyz
 wiflix.yachts
 coronavirus.zone
+xss.ls
+elektrum.sbs
```

Comment:
not active

![image](https://github.com/user-attachments/assets/3ef4f860-a7a2-4540-9296-ef9f497f0395)

Reply:
```
pyfunceble -a -d xss.ls

    ██████╗ ██╗   ██╗███████╗██╗   ██╗███╗   ██╗ ██████╗███████╗██████╗ ██╗     ███████╗
    ██╔══██╗╚██╗ ██╔╝██╔════╝██║   ██║████╗  ██║██╔════╝██╔════╝██╔══██╗██║     ██╔════╝
    ██████╔╝ ╚████╔╝ █████╗  ██║   ██║██╔██╗ ██║██║     █████╗  ██████╔╝██║     █████╗
    ██╔═══╝   ╚██╔╝  ██╔══╝  ██║   ██║██║╚██╗██║██║     ██╔══╝  ██╔══██╗██║     ██╔══╝
    ██║        ██║   ██║     ╚██████╔╝██║ ╚████║╚██████╗███████╗██████╔╝███████╗███████╗
    ╚═╝        ╚═╝   ╚═╝      ╚═════╝ ╚═╝  ╚═══╝ ╚═════╝╚══════╝╚═════╝ ╚══════╝╚══════╝
    


Subject                                                                                              Status      Source     Expiration Date   HTTP Code  Checker      
---------------------------------------------------------------------------------------------------- ----------- ---------- ----------------- ---------- -------------
xss.ls                                                                                               ACTIVE      WHOIS      12-jul-2025       Unknown    AVAILABILITY
```